recipes/openblas/all/conanfile.py

```diff
@@ -0,0 +1,81 @@
+from conans import ConanFile, CMake, tools
+import sys
+import os
+
+
+class OpenBLAS(ConanFile):
+    name = "openblas"
+    version = "0.3.7"
+    license = "BSD 3-Clause"
```

Comment:
```suggestion
    license = "BSD-3-Clause"
```

https://spdx.org/licenses/BSD-3-Clause.html